File: code/cfg/Task1.java
package cfg;
public class Task1 extends Task {
public final static int __TYPE_ID__ = 1;
public final int getTypeid() { return __TYPE_ID__; }
public final float x;
public Task1(CSVStream fs) {
super(fs);
this.x = fs.getFloat();
}
}

File: code/cfg/Task1.cs
using System;
namespace cfg{
public class Task1 : Task {
public const int __TYPE_ID__ = 1;
public int GetTypeid() { return __TYPE_ID__; }
public readonly float x;
public Task1(CSVStream fs) : base(fs) {
this.x = fs.GetFloat();
}
}
}